src/core/server/saved_objects/migrationsv2/model/model.test.ts

```diff
@@ -1529,18 +1539,28 @@ describe('migrations v2 model', () => {
         hasTransformedDocs: false,
         progress: createInitialProgress(),
       };
-      test('TRANSFORMED_DOCUMENTS_BULK_INDEX should throw a throwBadResponse error if action failed', () => {
```

Comment:
removed this test since it was testing our retry logic for retryable errors which is already covered in other places